pkgs/development/python-modules/fpdf2/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fonttools
+}:
+
+buildPythonPackage rec {
+  pname = "fpdf2";
+  version = "2.7.6";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-EW6zcmYNPaVaoCAMCd3KjNCdriAITofwjryCyb/oRm4=";
+  };
+
+  # tests use filesystem
+  doCheck = false;
+
+  propagatedBuildInputs = [ fonttools ];
+
+  meta = {
+    homepage = "https://github.com/py-pdf/fpdf2";
+    description = "Simple PDF generation for Python";
+    license = lib.licenses.lgpl3;
```

```suggestion
    license = lib.licenses.lgpl3Only;
```
I guess.
https://github.com/py-pdf/fpdf2/blob/bc44876c9e2b2a96d77ac4d135be01620332c65d/setup.cfg#L17

---

(here, foo is just a placeholder)
There are certain packages which allow people to distribute a package under license foo v3 or higher, for those you'd do foo3Plus
If only v3 is allowed you'd do foo3Only

foo3 by itself might be a deprecated license, and should be replaced with one the two above

---

The gpl license is complicated, and I check it with the following steps.
Using gpl3 as an example
1. search for the string `later` and check if the source code contains the phrase `version 3 of the License, or (at your option) any later version`.
There may be lines in the header of each file that indicate the license.
If the above statement exists, it is gpl3Plus.
2. If the software is a python module, look for the license in setup.py, setup.cfg, or pyproject.toml.
If it says something like `GPL3+`, it is gpl3Plus.
3. if you don't see any `or later` or `+`, set it to gpl3Only to avoid applying the wrong license.